index.d.ts

```diff
@@ -89,7 +89,7 @@ declare interface VuetifyObject {
   dark: boolean
   theme: VuetifyTheme | false
   options: VuetifyOptions
-  goTo: (target: string | number | HTMLElement | Vue, options?: VuetifyGoToOptions) => void
+  goTo: (target: string | number | HTMLElement | Vue, options?: VuetifyGoToOptions) => Promise
```

Comment:
`Promise` needs a type argument, in this case it's the same as `target` so I'd make `goTo` generic as well:

```ts
goTo: <T extends string | number | HTMLElement | Vue>(target: T, options?: VuetifyGoToOptions) => Promise<T>
```